text/entities/0264-resource-and-entities.md

```diff
@@ -0,0 +1,811 @@
+- **Embed fully Entity in Resource.** This was rejected because it makes it easy/trivial for Resource attributes and Entities to diverge.  This would prevent the backwards/forwards compatibility goals and also require all participating OTLP users to leverage entities. Entity should be an opt-in / additional feature that may or may not be engaged with, depending on user need.
+- **Re-using resource detectoin as-is** This was reject as not having a viable compatibility path forward.  Creating a new set of components that can preserve existing behavior while allowing users to adopt the new functionality means that users have better control of when they see / change system behavior, and adoption is more obvious across the ecosystem.
+
+## Future Posibilities
+
+This proposal opens the door for addressing issues where an Entity's lifetime does not match an SDK's lifetime, in addition to providing a data model where mutable (descriptive) attributes can be changed over the lifetime of a resource without affecting its idnetity.  We expect a follow-on OTEP which directly handles this issue.
+
+## Use Cases
+
+Below are a set of use cases to help motivate this design.
+
+### SDK - Multiple Detectors of the same Entity type
+
+Let's consider the interaction of the SDK in the presence of multiple registered
+entity detectors:
+
+```mermaid
+flowchart LR
+    SDK["`**SDK**`"] -->|OTLP| BACKEND["`**Backend**`"]
+    SDK -.- RC((Resource Provider))
+    RC -.- OTEL_DETECTOR((OpenTelemetry Default Resource Detection))
+    RC -.- GCP_DETECTOR((Google Cloud Specific Resource Detection))
+    GCP_DETECTOR -. Detects .-> GCE{{gcp.gce}}
+    GCP_DETECTOR -. Detects .-> GCPHOST{{"host (gcp)"}}
+    OTEL_DETECTOR -. Detects .-> HOST{{"host (generic)"}}
+    OTEL_DETECTOR -. Detects .-> PROCESS{{process}}
+    OTEL_DETECTOR -. Detects .-> SERVICE{{service}}
+```
+
+Here, there is a services running on the Google Compute Engine. The user
+has configured a Google Cloud specific set of entity detectors.  Both the
+built in OpenTelemetry detection and the configured Google Cloud detection
+discover a `host` entity.
+
+The following outcome would occur:
+
+- The resulting resource would have all of the following entities: `host`, `process`, `service` and `gcp.gce`
+- The user-configured resource detector would take priority over built in: the `host` defined from the Google Cloud detection would "win" and be included in resource.
+  - This means `host.id` e.g. could be the id discovered for GCE VMs.  Similarly for other cloud provider detection, like Amazon EC2 where VMs are given a unique ID by the Cloud Provider, rather than a generic machinne ID, e.g.
```

Comment:
```suggestion
  - This means `host.id` e.g. could be the id discovered for GCE VMs.  Similarly for other cloud provider detection, like Amazon EC2 where VMs are given a unique ID by the Cloud Provider, rather than a generic machine ID, e.g.
```